packages/api-clients/open-api/purposeApi.yml

```diff
@@ -415,6 +415,53 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+    patch:
+      parameters:
+        - $ref: "#/components/parameters/CorrelationIdHeader"
+      tags:
+        - purpose
+      operationId: PATCH_updatePurpose
+      description: Updates a Purpose if not yet activated
```

Comment:
Maybe we could add in the description the fact that this method is "temporary", just for m2m use, and it will replace the POST update?

Reply:
I meant that this naming with `PATCH_` is temporary, and in the future we will remove the corresponding POST endpoint and the prefix